code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -96,9 +114,10 @@
 
 /obj/item/storage/belt/utility/chief
 	name = "advanced toolbelt"
-	desc = "Holds tools, looks snazzy"
+	desc = "Holds tools, looks snazzy, and fits nicely into a bag"
```

Comment:
You could possibly add this extra desc stuff to examine on base belts if they're storable

something like 
```
/obj/item/storage/belt/examine()
    . = ..()
    if(storable)
        . += "<span class='notice'>It looks like it'd fit nicely into a bag.</span>"
```

This is up to you, though, the individual flavor text is nice too.

Reply:
I like the custom ones to be honest, probably gonna stick with those.

Reply:
Looks good. On that note, you might want to put a period on the end of it, then :P